server/datastore/mysql/migrations/tables/migration_test.go

```diff
@@ -0,0 +1,86 @@
+// Package tables holds fleet table migrations.
+//
+// Migrations can be tested with tests following the following format:
+//
+//	$ cat 20220208144831_AddSoftwareReleaseArchVendorColumns_test.go
+//
+//	[...]
+//	func TestUp_20220208144831(t *testing.T) {
+//		// Apply all migrations up to 20220208144831 (name of test), not included.
+//		db := applyUpToPrev(t)
+//
+//		// insert testing data, etc.
+//
+//		// The following will apply migration 20220208144831.
+//		applyNext(t, db)
+//
+//		// insert testing data, verify migration.
+//	}
+package tables
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO(lucas): I'm copy pasting some of the mysql functionality methods here
+// otherwise we have import cycle errors.
```

Comment:
Maybe we can have a `server/datastore/mysql/mysqltest` package where these live?

Reply:
We are all on the same page :) https://github.com/fleetdm/fleet/pull/4112#discussion_r803024307